rocketbelt/components/buttons/rocketbelt.buttons-dropdown.js

```diff
@@ -51,23 +51,10 @@
     }
 
     document.addEventListener('click', (e) => {
-      const path = e.path;
-      const pathLen = path.length;
-      let shouldClose = true;
-
-      for (let j = 0; j < pathLen; j++) {
-        const el = path[j];
-
-        if (el.classList && (el.classList.contains('button-dropdown_content') || el.classList.contains('button-dropdown_icon') || el.classList.contains('button-dropdown_trigger'))) {
-          shouldClose = false;
-          break;
-        }
-      }
-
-      if (shouldClose) {
+      const el = $(e.target);
```

Comment:
Can you factor `$` out of this? Trying to minimize jQ dependencies and we can pull off the next line without it.